drivers/utils/driver_utils.bal

```diff
@@ -17,14 +17,21 @@
 import ballerina/http;
 import ballerina/lang.runtime;
 import ballerina/log;
+import ballerina/task;
 import ballerina/tcp;
 import ballerina/time;
 import ballerina/uuid;
 
 http:Client hubClient = check new ("localhost:9090"); //ToDo: Remove
```

Comment:
What are these hard coded values? Can u please remove them too? I know it's not related to this PR, but let's do please.

Reply:
When those variables are initialized to Nil ballerina gives the error "client resource access action is only allowed on client objects(BCE4027)". Have to remove those with a refactoring.